frameworks/Swift/perfect/benchmark_config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "framework": "perfect",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "Perfect",
+      "language": "Swift",
+      "orm": "Raw",
+      "platform": "Perfect",
+      "webserver": "Perfect",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "Perfect",
+      "notes": ""
+    },
+    "mysql": {
+      "update_url": "/updates?queries=",
+      "query_url": "/queries?queries=",
```

Comment:
You can add these two urls to the default test and have the default test do plaintext, json, update, and query

Reply:
I probably am not understanding how the benchmark-config works, but I was under the assumption that the default would hit perfect while each other would hit perfect-DB. If I added queries to perfect (which doesn't have a db) wouldn't it just fail?

Reply:
You'd add the DB to that config as well and have all routes registered in `/Perfect/main.swift`. The database connection can still be created when the server is started, but if unused in the `/plaintext`, `/json` routes it shouldn't have any effect on performance. A lot of tests will combine the plaintext and json results with the first db variant and then do a second db variant in another config.. but when you only have one it should look something like this: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Lua/lapis/benchmark_config.json

Reply:
Thanks. I have included 3 dbs now though. I wrote it to look like the express benchmark_config. Should I still make changes to it now that there are 3 db variants?

Reply:
Oh cool, I hadn't noticed. This is fine then as it matches what we do with gemini. Thanks!

Reply:
Awesome! Thanks!